test/test__formUI_status_test.py

```diff
@@ -39,6 +41,50 @@ def add_two_widgets(self):
         form.addWidget(QtWidgets.QLabel('test label'), 'Label: ', 'label')
         form.addWidget(QtWidgets.QCheckBox('test checkbox'), 'CheckBox: ', 'checkBox')
 
+    def set_state(self, ii):
```

Comment:
I added the docstring, thanks for reminding me.

I use `i` to distinguish from the imaginary unit for code transferability in other languages and to avoid single-character variables. What do you normally use?

Reply:
@DanicaSTFC told me that she used `ii` and `jj` when there was a possible confusion with the imaginary unit `i`. In this case we don't have such confusion, so the double letter is not required. 

Throughout our code we use single letter `i`, `j` and `k` as running indices normally.

Reply:
I changed `ii` to `i`.